### ISSUE TITLE
Importing circuit_extract fails

### ISSUE THREAD

**Issue by tbabej** [Quantomatic/pyzx#7]
This comes up when one attempts to execute the test suite:

```
>>> from pyzx.extract import circuit_extract
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'circuit_extract' from 'pyzx.extract'
```

since the definition of `circuit_extract` method is commented out.

**Comment by tbabej** [comment]
Same applies for `greedy_cut_extract`.

**Comment by johnie102** [comment]
Thank you for reporting this. The tests should have indeed been updated to reflect their deprecation.
Fixed by 069b88cd16b82be07dc0c4d69a83d7257e586580

**Comment by johnie102** [comment]
Ah yeah, I merged your pull request. Thanks!
